services-manager/src/main/resources/application-examplelibs.yaml

```diff
@@ -17,23 +17,22 @@ spring:
   profiles:
     include:
       - eureka
-      - example-config
+      - examplelibs-config
 
 # Spring profile dependency resolution forces this two-profile setup
 # `example` profile must forcibly override its dependencies as spring.profiles.include properties can no longer be overridden
 # So, include `example-config` along with dependencies and force correct override order to happen there
 ---
 
 spring:
-  profiles: example-config
+  profiles: examplelibs-config
```

Comment:
naming convention should be example-libs-config

Reply:
I would prefer example-config. It describes a state.